src/Event/ContaoCoreEvents.php

```diff
@@ -60,7 +60,7 @@
     public const BACKEND_MENU_BUILD = 'contao.backend_menu_build';
 
     /**
-     * The contao.slug_valid_characters event is triggered when the options for the valid characters setting for aliases are generated.
+     * The contao.slug_valid_characters event is triggered when the valid slug characters option is generated.
```

Comment:
I think this should be plural “options are generated”

Reply:
Are there several valid slug characters options?

Reply:
Yes, by default there are four options: unicode lowercase, unicode, ASCII lowercase and ASCII.